ZFSin/zfs/include/sys/zvol.h

```diff
@@ -76,6 +76,9 @@ typedef struct zvol_state {
 	/* 'rdiskX' name, use [1] for diskX */
 	uint8_t zv_target_id;
 	uint8_t zv_lun_id;
+	/* Variables involved to LUN reset mechanism. */
+	uint32_t zv_requests_processing_count; /* How meny SRBs are processing*/
```

Comment:
zv_requests_processing_count is truly not used for something significant. just inc and dec without any special processing if 0 or not so we should remove that.